source/sdk/threading/native/ThreadWin32.ooc

```diff
@@ -45,31 +37,26 @@ version(windows) {
 					false
 			}
 		}
-
 		cancel: func -> Bool {
 			false
 			//this alive?() && TerminateThread(this handle, 0)
 			//TODO Find a better way to terminate Win32 threads, if any
 		}
-
 		alive?: func -> Bool {
 			result := WaitForSingleObject(handle, 0)
 
 			// if it's equal, it has terminated, otherwise, it's still alive
 			result != WAIT_OBJECT_0
 		}
-
 		_currentThread: static func -> This {
-			thread := This new(func {})
+			thread := This new(func)
```

Comment:
The result should be the same, but magic doesn't like empty function definitions.